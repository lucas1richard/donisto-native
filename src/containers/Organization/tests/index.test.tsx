import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Organization, IOrganizationProps } from '../index';

describe('<Organization />', () => {
  const renderComponent = (props: IOrganizationProps) => toJson(shallow(<Organization {...props} />));
  it('Expect to have unit tests specified', () => {
    expect(renderComponent({
      organizationUuid: 'string',
      activeOrg: { name: 'any', uuid: 'uuid' },
    })).toMatchSnapshot();
  });
  it('Expect to have unit tests specified', () => {
    expect(renderComponent({})).toMatchSnapshot();
  });
});
