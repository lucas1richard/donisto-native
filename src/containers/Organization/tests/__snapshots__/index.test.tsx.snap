// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Organization /> Expect to have unit tests specified 1`] = `
<Component
  style={
    Object {
      "padding": 12,
    }
  }
>
  <H1
    color="default"
  >
    Organization
  </H1>
  <H2
    color="default"
  >
    any
  </H2>
  <Button
    color="secondary"
    onPress={[Function]}
    title="Switch to Donor Mode"
    variant="contained"
  />
</Component>
`;

exports[`<Organization /> Expect to have unit tests specified 2`] = `
<Component
  style={
    Object {
      "flex": 1,
    }
  }
>
  <ScrollViewMock>
    <Component
      style={
        Object {
          "padding": 12,
        }
      }
    >
      <H1
        color="default"
      >
        You're not associated to any organization yet
      </H1>
      <Connect(RegisterNewOrg) />
      <Button
        color="secondary"
        onPress={[Function]}
        title="Switch to Donor Mode"
        variant="contained"
      />
    </Component>
  </ScrollViewMock>
</Component>
`;
